pandas/tests/frame/test_missing.py

```diff
@@ -561,8 +561,13 @@ def test_interp_various(self):
         assert_frame_equal(result, expected)
 
         result = df.interpolate(method='cubic')
-        expected.A.loc[3] = 2.81621174
-        expected.A.loc[13] = 5.64146581
+        import scipy
+        if scipy.__version__ >= LooseVersion('0.19.0'):
+            expected.A.loc[3] = 2.81547781
```

Comment:
also can you add a comment here (1-liner), on what qualitatively changed (adding the link is ok too).

imagine you are  future reader and you have no idea why this was done....